test/crossbrowser.conf.js

```diff
@@ -19,6 +21,14 @@ exports.config = Object.assign(base, {
     // Picked up by cbt_tunnels.
     baseUrl: 'http://local',
 
+    // Bail after the first test failure. The cost of running many tests is too
+    // high, and it takes too damn long.
+    bail: 1,
+    // Up the test timeout to 60s.
```

Comment:
120s?

And isn't this just a milliseconds conversion?

Reply:
Woops! Yeah, it is, and yes it should be 120 seconds.

Reply:
Fixed in https://github.com/unravelin/ravelinjs/pull/26/commits/da4d25defd7ef53c286c4aabb652a49014158960